src/legacyhaxelib/Data.hx

```diff
@@ -67,7 +67,7 @@ class Data {
 	public static var DOCXML = "haxedoc.xml";
 	public static var REPOSITORY = "files";
 	public static var alphanum = ~/^[A-Za-z0-9_.-]+$/;
-	static var LICENSES = ["GPL","LGPL","BSD","Public","MIT"];
+	static var LICENSES = ["GPL","LGPL","BSD","Public","MIT", "Apache", "MPL"];
```

Comment:
The legacy server is read only so no new libraries will be uploaded, so there is no need to change the license list there

Reply:
fixed